.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 language: go
 sudo: required
 go:
-- 1.7
+- 1.7.x
```

Comment:
@skonefal - I assume that we should be running latest version from chosen branch anyway and this approach allows to do so. If we were to add similar automation to vagrant provisioning then we would have to code it or use `gimme` as Travis does.
Side note: 1.7.6 was not available on Travis last night.